backend/cognito/cognito.template.yml

```diff
@@ -22,6 +22,12 @@ Parameters:
     Type: String
     Default: ""
     Description: The ARN of the permissions boundary to apply when creating IAM roles [Secure Pipelines]
+  NotifyApiKey:
+    Type: String
+    Default: please-provide-notify-api-key
+  SecurityCodeTextMessageTemplate:
+    Type: String
+    Default: ac71249b-efba-4e05-911d-83648caa8dd5
```

Comment:
This needs to be a parameter store param named `/self-service/cognito/security-code-text-message-template`

Reply:
addressed